packages/cli/tests/e2e/bot/NotificationBotHappyPathCommon.ts

```diff
@@ -34,7 +34,7 @@ export function happyPathTest(runtime: "dotnet" | "nodejs"): void {
 
     it("Provision Resource: app service hosted notification", async function () {
       const cmd =
-        runtime === "nodejs"
+        runtime === "node"
```

Comment:
Nit: I remember there's an enum, how about using enum here instead of a plain string `node`?

Reply:
Defined in fx-core. Is it ok to import from another package?